content/livesync/quickstart.textile

```diff
@@ -1,7 +1,241 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+This quickstart will provide you with a set of instructions to get an example running locally on your machine. Following this, you'll cover each component of LiveSync found within the code of the example.
+
+h2(#example). The example
+
+In this section you will cover all the steps required to get an existing example running locally on your machine.
+
+h3(#step-0). Clone the repository
+
+Clone the "models repository":https://github.com/ably-labs/models/ and navigate to the @livecomments@ directory within @examples@. The @livecomments@ directory houses the project used for this example.
+
+h3(#step-1). Create an Ably account
+
+"Sign up":https://ably.com/signup for a free account to get your own API key. Ensure that your API key includes the @subscribe@ and @publish@ "capabilities":/auth/capabilities.
+
+h3(#step-2). Update environment variables
+
+Set up environment variables by copying them from the template:
+
+```[sh]
+ cp env.example env.local
+```
+
+Update your @env.local@ file with the following:
+
+```
+POSTGRES_URL="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_PRISMA_URL="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_URL_NON_POOLING="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_USER=postgres
+POSTGRES_HOST=localhost
+POSTGRES_PASSWORD=postgres
+POSTGRES_DATABASE=postgres
+SESSION_SECRET=<SOME_SECRET>
```

Comment:
I would just set this to some default value, since we're running locally it doesn't matter what it is, perhaps just call it `somesecret`